hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -211,6 +211,23 @@ public class HoodieWriteConfig extends HoodieConfig {
           + " optimally for common query patterns. For now we support a build-in user defined bulkinsert partitioner org.apache.hudi.execution.bulkinsert.RDDCustomColumnsSortPartitioner"
           + " which can does sorting based on specified column values set by " + BULKINSERT_USER_DEFINED_PARTITIONER_SORT_COLUMNS.key());
 
+  public static final ConfigProperty<String> BULKINSERT_ROW_IDENTIFY_ID = ConfigProperty
+      .key("hoodie.bulkinsert.row.writestatus.id")
+      .noDefaultValue()
+      .withDocumentation("The unique id for each write operation, HoodieInternalWriteStatusCoordinator will use "
```

Comment:
why we need set this config？ how about use uuid directly.

Reply:
This is an internal configure used by `HoodieDataSourceInternalBatchWrite` and `HoodieDataSourceInternalWriter` to pass `writeStatuses` to the clustering job.
At first I thought setting this in `DataSourceInternalWriterHelper` like `INSTANT_TIME_OPT_KEY`, but it's difficult for `ClusteringExecutionStrategy` to access this as package `hudi-client` cannot access `hudi-spark-datasource`, So I keep this config in `HoodieWriteConfig`.